CMakeLists.txt

```diff
@@ -183,6 +195,10 @@ if(NOT NUMA_LIBRARY)
     message(FATAL_ERROR "NUMA library not found")
 endif()
 
+find_package(nng)
+find_package(flatbuffers)
+find_package(libuv)
```

Comment:
This will not work currently in our flow in CI, as we don't install any of these into the system.

We bring them in straight from CPM build tree.

There are requests for us to support either flows, but we aren't there yet.

This should be removed for now.

Reply:
Ok I will remove them.